runtime/src/snapshot_minimizer.rs

```diff
@@ -346,8 +347,11 @@ impl<'a> SnapshotMinimizer<'a> {
         if aligned_total > 0 {
             shrink_in_progress = Some(self.accounts_db().get_store_for_shrink(slot, aligned_total));
             let new_storage = shrink_in_progress.as_ref().unwrap().new_storage();
+
+            let accounts = [(slot, &keep_accounts[..])];
```

Comment:
snapshot minimizer is another type of shrink. source and target slot are both `slot`. `StorableAccountsBySlot`handles this use case fine.